packages/wallet-sdk/src/features/sub-accounts/types.ts

```diff
@@ -0,0 +1,34 @@
+import { Hex } from 'ox';
+import { SignReturnType } from 'viem/accounts';
+import type { SignReturnType as WebAuthnSignReturnType } from 'webauthn-p256';
+
+export type SubAccountSigner = {
+  type: 'webAuthn'; // TODO: add local support
+  /**
+   * Signs a raw payload
+   * @param rawPayload - The raw payload to sign
+   * @returns {Hex} Signed payload
+   */
+  getSigner: () => Promise<
```

Comment:
can we consolidate the types here? Just like a Viem account? Or what is webauthn different than normal

Reply:
@jxom any take? :D

Reply:
I've updated this to resolve to a viem account, this removes the need to getAddress and type methods

new interface 

```tsx
getSigner: () => Promise<LocalAccount|WebAuthnAccount>
```

Reply:
> can we consolidate the types here? Just like a Viem account? Or what is webauthn different than normal

The return signature types are different. If we want to keep this generic we'll likely need our sign function to take a Hex and resolve to a Hex and then internally map to the various account types i.e. Local or WebAuthn